posthog/api/property_definition.py

```diff
@@ -474,3 +476,10 @@ def get_object(self):
     @extend_schema(parameters=[PropertyDefinitionQuerySerializer])
     def list(self, request, *args, **kwargs):
         return super().list(request, *args, **kwargs)
+
+    def destroy(self, request: request.Request, *args: Any, **kwargs: Any) -> response.Response:
+        instance = self.get_object()
+        self.perform_destroy(instance)
+        # Casting, since an anonymous use CANNOT access this endpoint
+        report_user_action(cast(User, request.user), "property definition deleted", {"name": instance.name})
```

Comment:
could you add type in addition to "name" as well for easier usage later, it's helpful to know which 3 it is from https://github.com/PostHog/posthog/blob/016c49cb41d9dd708b55de287bb466a4996303c2/posthog/models/property_definition.py#L31-33

Reply:
Done